frontend/src/components/layout/SideBar.tsx

```diff
@@ -1,7 +1,6 @@
 /*
-Copyright (c) 2022 Volkswagen AG
-Copyright (c) 2022 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
-Copyright (c) 2022 Contributors to the Eclipse Foundation
+Copyright (c) 2024 Volkswagen AG
```

Comment:
Likely autoupdate: Why did you update the Copyright to 2024 and remove Fraunhofer?